### PR TITLE
Pete - final changes to initial version of Capstone2 project website

### PR DESCRIPTION
**Pull request by redpath2312**
Final tweaks meaning can merge into Master. May expand website in future eg About me section, and add more projects later on.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,54 +32,64 @@
       <div class="row flex-lg-row-reverse align-items-center g-5 py-5">
         <div class="col-10 col-sm-8 col-lg-6">
           <img src="./images/devbackgroundpic.jpg" class="d-block mx-lg-auto img-fluid" alt="Bootstrap Themes" width="700" height="500" loading="lazy">
+          <p class= "heroattribution">Image by <a href="https://www.freepik.com/free-vector/gamer-room-illustration_13558844.htm#query=simple%20game%20developer%20home%20desk&position=31&from_view=search&track=ais">Freepik</a></p>
```
Comment (algren123) [inline index.html]: heroattribution -> heroAttribution

try and use camel casing when setting classes

File: index.html [hunk 2]
```diff
@@ -32,54 +32,64 @@
...
-  
+    <div class="album py-5">
+      <div class="container">  
         <div class="row row-cols-1 row-cols-sm-2 row-cols-md-3 g-3">
+
           <div class="col">
             <a href = "./webdev.html">   
             <div class="card shadow-sm">                 
-              <img src = "images/programming-background-collage.jpg" alt = "web dev background">           
+              <img class= "mainimages" src = "images/programming-background-collage.jpg" alt = "web dev background">
```
Comment (algren123) [inline index.html]: same camel casing here

File: index.html [hunk 3]
```diff
@@ -32,54 +32,64 @@
...
-                <p class="card-text">Web Development - Projects and pages that i've created so far. </p>
-                <!-- Image by <a href="https://www.freepik.com/free-photo/programming-background-collage_34089161.htm#query=website%20development&position=36&from_view=search&track=ais">Freepik</a>-->
-               </div>
-            </div>
-          </a></div>
+                <p class="card-text">Web Development - Projects and pages that i've created so far. </p>                                
+               </div>               
+            </div>            
+            </a>
+          <p class = "imageattribution">Image by <a href="https://www.freepik.com/free-photo/programming-background-collage_34089161.htm#query=website%20development&position=36&from_view=search&track=ais">Freepik</a></p>
```
Comment (algren123) [inline index.html]: camel case here

File: index.html [hunk 4]
```diff
@@ -32,54 +32,64 @@
...
+          </div>
+          <!-- Image by <a href="https://www.freepik.com/free-photo/programming-background-collage_34089161.htm#query=website%20development&position=36&from_view=search&track=ais">Freepik</a>-->
+
           <div class="col">
+            <a href = "./gamedev.html">   
             <div class="card shadow-sm">
-              <img src = "images/game dev background.jpg" alt = "game dev background">
-  
+              <img class= "mainimages" src = "images/game dev background.jpg" alt = "game dev background">
```
Comment (algren123) [inline index.html]: camel case here

File: css/style.css [hunk 5]
```diff
@@ -1,3 +1,70 @@
...
+{
+    .mainimages {
+        height: 175px;
+    }
+}
+@media(min-width: 301px) and (max-width: 575px)
+{
+    .mainimages {
+        height:175px;
+      /*
```
Comment (algren123) [inline css/style.css]: commented code

**Review by algren123 — CHANGES_REQUESTED** [review]
1. camel casing the classes, ids etc. or use dashes (mainImages vs main-images)
2. rewriting the CSS media queries so they encapsulate all the class logic into a single style.
3. rework/research Bootstrap columns and breakpoints (https://getbootstrap.com/docs/5.3/layout/columns/#column-breaks)

**Comment by redpath2312** [comment]
The camel casing and media query rework done locally easy enough.
I can't remember the end goal of the column breaks and what the specific functionality /layout we were trying to get with them though i did find it finnicky, so guessing it was trying to make it simpler!?

**Pull request merged**